torch/csrc/jit/passes/alias_analysis.h

```diff
@@ -179,6 +181,9 @@ class AliasDb {
   // or does the element contain an element for which that's true
   bool cannotCheckAliasContainment(const Value* elem) const;
 
+  // Is this a value which will not alias
```

Comment:
Please add a more descriptive comment. There's a lot of subtlety because `nonAliasingValue` is not the same as `!shouldAnnotate()`, so it would be valuable to explicitly explain what the difference between those two things is

Reply:
The more descriptive comment in is the cpp comment, and this is a private method.